src/shared/ui/MarkdownView.tsx

```diff
@@ -32,14 +32,22 @@ declare module 'react-native-markdown-display' {
 }
 
 const MarkdownView: FC<Props> = ({ content, markdownStyle, rules }) => {
+  const [processedContent, setProcessedContent] = useState<string>('');
+
+  useEffect(() => {
+    preprocessImageLinks(content).then(result => {
+      setProcessedContent(result);
+    });
+  }, [content]);
```

Comment:
`preprocessImageLinks` utilizes [Image.getSize](https://reactnative.dev/docs/image#getsize) that **downloads** the images first and then calculates the size. 
What if there is no Internet connection? Or, what if the Internet Connection is poor? I believe `processedContent` could be empty for a while in this case